cdap-ui/server/express.js

```diff
@@ -61,11 +62,31 @@ const getExpressStaticConfig = () => {
     maxAge: '1y'
   };
 };
+
+function getFaviconPath(uiThemeConfig) {
+  let faviconPath = DIST_PATH + '/assets/img/favicon.png';
+  let themeFaviconPath;
+  let themeFaviconPathType = objectQuery(uiThemeConfig, ['content', 'favicon', 'type']);
+  let themeFaviconPathUrl = objectQuery(uiThemeConfig, ['content', 'favicon', 'arguments', 'url']);
+  if (themeFaviconPathType === 'link' && themeFaviconPathUrl) {
```

Comment:
what's another accepted value for type?

Reply:
For other images, you can also have type as `inline` and provide a base64 encoded image, and `img src` will pick up that, but seems like `serve-favicon` doesn't allow that. So l can just change the theme property to be `faviconPath` which takes in a string for path. That'll make it simpler.